### ISSUE TITLE
pagination for pages

### ISSUE THREAD

**Issue by fabianmoronzirfas** [basiljs/basiljs.github.io#14]
Jekyll does not offer pagination for pages as menitioned by @trych in #13 . What could be done is keep a list all the pages we have in the gallery and compare them with each other. This should be done with liquid so we don't need to update something by hand.

Another approach would be generating an json object within the page and use JS to build those links. Or other magic

**Comment by trych** [comment]
When I was messing around with Jekyll last summer, I put together some sort of pagination feature. The problem is that I neither remember how Jekyll works, nor how I set this up. Here is the relevant snippet:

```html
<li><a id="prev" href=
        {% if page.previous %}
          "{{ site.baseurl }}{{ page.previous.url }}"
        {% else %}
          "{{ site.baseurl }}/{{ page.collection }}.html"
        {% endif %}>&larr;</a> <a id="next" href=
        {% if page.next %}
          "{{ site.baseurl }}{{ page.next.url }}"
        {% else %}
          "{{ site.baseurl }}/{{ page.collection }}.html"
        {% endif %}>&rarr;{{ page.collection.label }}</a>
</li>
```

The problem is, I don't quite remember where page.next and page.previous come from. Just thought I drop this here, maybe it helps.

**Comment by fabianmoronzirfas** [comment]
Thanks. I also saw this while researching. I think this only works for posts and not for pages. But will take a look into the pagination Jekyll offers

**Comment by trych** [comment]
I don't think it was posts, but a "projects" category that I defined myself. However, this might just be the same thing as posts, I am not sure anymore, too long ago.
